<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BacktesterManager</class>
 <widget class="QWidget" name="BacktesterManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BacktesterManager</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>441</width>
     <height>541</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>交易策略</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>合约代码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>K线周期</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>开始日期</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_8"/>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>结束日期</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_11"/>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>手续费率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_10"/>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>交易滑点</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_9"/>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>合约乘数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_12"/>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>价格跳动</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_13"/>
         </item>
        </layout>
       </item>
       <item row="9" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>回测资金</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_14"/>
         </item>
        </layout>
       </item>
       <item row="10" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>开始回测</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>下载数据</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>成交几率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>委托记录</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="13" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>每日盈亏</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>k线图表</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="14" column="0">
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>参数优化</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string>优化结果</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QFrame" name="frame_5">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>首个交易日</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>最后交易日</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_26">
             <item>
              <widget class="QLabel" name="label_33">
               <property name="text">
                <string>总交易日</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_34">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>盈利交易日</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_20">
             <item>
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>亏损交易日</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_25">
             <item>
              <widget class="QLabel" name="label_31">
               <property name="text">
                <string>起始资金</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_32">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_24">
             <item>
              <widget class="QLabel" name="label_29">
               <property name="text">
                <string>结束资金</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_30">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_23">
             <item>
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>总收益率</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_28">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_27">
             <item>
              <widget class="QLabel" name="label_35">
               <property name="text">
                <string>年化收益率</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_36">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_22">
             <item>
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>最大回撤</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_21">
             <item>
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>百分比最大回撤</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QFrame" name="frame_6">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEdit"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>431</width>
     <height>511</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCustomPlot" name="widget" native="true"/>
    </item>
    <item>
     <widget class="QCustomPlot" name="widget_2" native="true"/>
    </item>
    <item>
     <widget class="QCustomPlot" name="widget_3" native="true"/>
    </item>
    <item>
     <widget class="QCustomPlot" name="widget_4" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>BacktesterManager</receiver>
   <slot>startBacktest_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>-17</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startBacktest_clicked()</slot>
 </slots>
</ui>
